src/components/CurrentCityWeather/CurrentCityWeather.js

```diff
@@ -40,18 +47,34 @@ const CurrentCityWeather = () => {
       .catch((err) => {
         setError(err);
       });
-  }, [city]);
+  },[cityCoordinates],[]);
+
+
+  function setCoordinates(place) {
+    var latitude = place.geometry.location.lat();
+    var longitude = place.geometry.location.lng();
+    var coordinates = {lat: latitude, lon: longitude};
```

Comment:
It's best to use `const` variables wherever possible. When you need the variable to be mutable, `let` is usually better than `var`. To learn more check [this article](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/) out.